chessboard/script.js

```diff
@@ -0,0 +1,97 @@
+const board = document.querySelector(".board");
+const chess = document.querySelector(".chess");
+const bottom = document.createElement("div");
+const left = document.createElement("div");
+bottom.className = "bottom";
+left.className = "left";
+for (let i = 0; i < 8; i++) {
+  console.log(i, String.fromCharCode(i + 97));
+  const div = document.createElement("div");
+  const div1 = document.createElement("div");
+  const p = document.createElement("p");
+  const p1 = document.createElement("p");
+  p.textContent = String.fromCharCode(i + 97);
+  div.appendChild(p);
+  bottom.appendChild(div);
+  p1.textContent = 8 - i;
+  div1.appendChild(p1);
+  left.appendChild(div1);
+}
+chess.appendChild(bottom);
+chess.appendChild(left);
+console.log(board);
+for (let i = 0; i < 8; i++) {
+  const block = document.createElement("div");
+  block.className = "block";
+  for (let j = 0; j < 8; j++) {
+    const box = document.createElement("div");
+    box.className = "box";
+    box.id = `box` + (i + 1) + "" + (j + 1);
+    block.appendChild(box);
+  }
+  board.appendChild(block);
+}
+const check = (upper) => {
```

Comment:
The naming of the function can be better, using 'isValidMove' is a better choice 
Also, if you are returning true inside if block, then you can directly return false outside of the if condition without adding the else block

I have added an extra parameter id here to increase reusability of the function 
```
const isValidMove = (id) => {
  const row = Math.floor(id / 10);
  const col = id % 10;
  return row >= 1 && row <= 8 && col >= 1 && col <= 8;
};
```